registrations/api.py

```diff
@@ -142,7 +142,9 @@ def _get_message_signups(signups, signup_groups):
             responsible_signups = (
                 signup_group.responsible_signups or signup_group.signups.all()
             )
-            for signup in responsible_signups.exclude(email=None):
+            for signup in responsible_signups.exclude(
+                Q(email=None) | Q(attendee_status=SignUp.AttendeeStatus.WAITING_LIST)
```

Comment:
Pondering that should it be better to be explicit with the attendee_status here 🤔 Like if we would have a new status those would be included in the send as well - likely not going to happen but better to explicit imo 🧐 

`.exclude(email=None).filter(attendee_status=SingnUp.AttendeeStatus.ATTENDING)`

Reply:
This is a valid point - I changed it to the suggested explicit form.